### ISSUE TITLE
The rule `selector-pseudo-class-no-unknown` skips an unknown pseudo-class

### ISSUE THREAD

**Issue by in-in** [stylelint/stylelint#4220]
> Clearly describe the bug

The rule `selector-pseudo-class-no-unknown` skips an unknown pseudo-class  

> Which rule, if any, is the bug related to?

`selector-pseudo-class-no-unknown`

> What CSS is needed to reproduce the bug?

```css
li:last-child() {
  margin-bottom: 0;
}
```
or
```css
li:last-child) {
  margin-bottom: 0;
}
```

> What stylelint configuration is needed to reproduce the bug?

```json
{
  "rules": {
    "selector-pseudo-class-no-unknown": true
  }
}
```

> Which version of stylelint are you using?

`10.1.0`

> How are you running stylelint: CLI, PostCSS plugin, Node.js API?

CLI with `stylelint "**/*.css" --config myconfig.json`

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

No

> What did you expect to happen?

Warnings to be flagged

> What actually happened (e.g. what warnings or errors did you get)?

✔ No warnings

![192341566487260](https://user-images.githubusercontent.com/8797432/63527384-abd8cb00-c509-11e9-8cdb-0285060f19de.png)

**Comment by evilebottnawi** [comment]
It is invalid CSS syntax,stylelint doesn't support invalid syntax (because we can't parse broken CSS), you can create plugin for validate syntax
